src/transformers/generation_utils.py

```diff
@@ -743,6 +759,47 @@ def _merge_criteria_processor_list(
         default_list.extend(custom_list)
         return default_list
 
+    def compute_transition_beam_scores(
+        self,
+        sequences: torch.Tensor,
+        scores: Tuple[torch.Tensor],
+        beam_indices: torch.Tensor,
+        eos_token_id: int = None,
+        dummy=None,
```

Comment:
Is that necessary ?

Reply:
great catch! No this was for debugging haha